File: app/src/debug/java/com/sakuna63/tumbin/application/di/module/DebugApplicationModule.java
package com.sakuna63.tumbin.application.di.module;

import android.app.Application;

import io.realm.RealmConfiguration;

public class DebugApplicationModule extends ApplicationModule {

    public DebugApplicationModule(Application app) {
        super(app);
    }

    @Override
    RealmConfiguration realmConfiguration() {
        return new RealmConfiguration.Builder()
                .deleteRealmIfMigrationNeeded()
                .build();
    }
}

File: app/src/debug/java/com/sakuna63/tumbin/application/di/module/DebugApplicationModule.kt
package com.sakuna63.tumbin.application.di.module

import android.app.Application

import io.realm.RealmConfiguration

class DebugApplicationModule(app: Application) : ApplicationModule(app) {

    override fun realmConfiguration(): RealmConfiguration =
            RealmConfiguration.Builder().deleteRealmIfMigrationNeeded().build()
}
